arch/arm/soc/atmel_sam/common/arm_mpu_regions.c

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm/cortex_m/mpu/arm_mpu.h>
+
+#include "arm_mpu_mem_cfg.h"
+
+/* System Private Peripheral Bus */
+#define PPB_BASE  0xE0000000
+
+static struct arm_mpu_region mpu_regions[] = {
+	/* Region 0 */
+	MPU_REGION_ENTRY("FLASH_0",
+			 CONFIG_FLASH_BASE_ADDRESS,
+			 REGION_FLASH_ATTR(REGION_FLASH_SIZE)),
+
+	/* Region 1 */
+	MPU_REGION_ENTRY("RAM_0",
+			 CONFIG_SRAM_BASE_ADDRESS,
+			 REGION_RAM_ATTR(REGION_SRAM_0_SIZE)),
```

Comment:
If my above assertion is NOT correct, why don't we have a region for the REGION_SRAM1 stuff you put in the arm_mpu_mem_cfg.h? Not all memory will be able to be addressed.
This is probably moot though, I think we can just leverage the background map and get rid of this entirely.